### DISCUSSION TITLE
Decoder Process Killed error in Discord Python Bot programming

### DISCUSSION THREAD

**Discussion by LHaggot** [Rapptz/discord.py#9922] [Q&A]
I have little to beginner experience in coding and have been pretty confused with how Discord has their API setup for programming bots, but I have been following guides online and they have been seeming to work so far. I'm trying to make a bot that will join call and randomly record 5-10 second clips of audio and send it into the channel, so to have spontaneous memories random recorded when me and my friends are gaming.

The problem I'm having is that whenever `vc.stop_recording` is called, I get an infinite line of `Decoder Process Killed` in my terminal, and the bot crashes. I'm working inside Replit if that gives any more context, attached is a screenshot of my recording method and my callback method.

![Help](https://github.com/user-attachments/assets/b4c55a27-c194-400d-9307-e16b318c32ee)

I tried looking through the Opus API, the Pycord API, and the discord API and I still haven't been able to find what is printing `Decoder Process Killed` in the terminal.

**Comment by oiyl** [comment]
I found it in opus, but not in this repo for some reason.

https://github.com/Pycord-Development/pycord/blob/96180fbf9266505e419a0b91fb0c1df581f8a038/discord/opus.py#L563

opus's stop() function calls it, which appears to not exist in this repo for some odd reason. I'm not sure.

It's probably running `stop()` multiple times in a loop.
